packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/column.js

```diff
@@ -116,9 +114,36 @@ const Column = props => {
         width
     };
 
+    if (image) {
+        dynamicStyles.backgroundImage = bgImageStyle;
+        dynamicStyles.backgroundSize = backgroundSize;
+        dynamicStyles.backgroundPosition = backgroundPosition;
+        dynamicStyles.backgroundAttachment = backgroundAttachment;
+        dynamicStyles.backgroundRepeat = backgroundRepeat
+            ? 'repeat'
+            : 'no-repeat';
+    }
+
+    // Determine the containers width and optimize the image
+    useEffect(() => {
+        if (image && columnElement.current) {
+            setBgImageStyle(
+                `url(${resourceUrl(image, {
+                    type: 'image-wysiwyg',
+                    width: columnElement.current.offsetWidth,
+                    height: columnElement.current.offsetHeight,
+                    quality: 85,
+                    crop: false,
```

Comment:
Since width and height are present this will be converted to true

Reply:
I've fixed the logic within addImgOptMiddleware to handle this correctly